azurerm/resource_arm_function_app.go

```diff
@@ -681,6 +701,12 @@ func expandFunctionAppSiteConfig(d *schema.ResourceData) web.SiteConfig {
 		siteConfig.LinuxFxVersion = utils.String(v.(string))
 	}
 
+	if v, ok := config["cors"]; ok {
+		corsSettings := v.(interface{})
+		expand := azure.ExpandAppServiceCorsSettings(corsSettings)
```

Comment:
As this is now used by two resources might be more aptly named:
```suggestion
		expand := azure.ExpandWebCorsSettings(corsSettings)
```